scripts/build_example

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+readonly SCRIPTS_DIR="$(dirname "$0")"
+
+COMPILATION_MODE="opt"
+
+while getopts "hd" opt; do
+    case "$opt" in
+    h)
+        echo -e "Usage: ./scripts/build_example [-d] [example_name]
```

Comment:
```suggestion
        echo -e "Usage: $0 [-d] [example_name]
```

Reply:
Not sure this works correctly, please try it first :)

Reply:
Works!